src/KafkaFlow.UnitTests/BatchConsume/BatchConsumeMiddlewareTests.cs

```diff
@@ -2,94 +2,139 @@ namespace KafkaFlow.UnitTests.BatchConsume
 {
```

Comment:
What do you think about we create integration tests with batch produce/consume?

Reply:
Yes we must create